src/browser.coffee

```diff
@@ -58,7 +58,7 @@ CoffeeScript.load = (url, callback, options = {}, hold = false) ->
 # Activate CoffeeScript in the browser by having it compile and evaluate
 # all script tags with a content-type of `text/coffeescript`.
 # This happens on page load.
-runScripts = ->
+CoffeeScript.runScripts = ->
```

Comment:
What prompted exposing `runScripts()`?

Reply:
I was thinking that the browser compiler (or the ES module version, at least) shouldn’t have side effects merely by being imported into the page. Like if the user wants to register an event handler for CoffeeScript to transpile and evaluate all `<script type="text/coffeescript">` blocks, the user should have to do so explicitly; it shouldn’t happen automatically. Especially now when those blocks could be either Script or Module code (i.e. they could use `import` statements) and the compiler doesn’t know that from just the `<script type="text/coffeescript">` tag.

I just checked and apparently I’m still having `runScripts` run on load rather than requiring registering, so I need to fix this. It’s attached to `CoffeeScript` so that it can be exported (see the `const { VERSION, compile, eval: evaluate, load, run, runScripts } = CoffeeScript;` line).